modules/concurrency/include/hephaestus/concurrency/spinner.h

```diff
@@ -11,24 +11,37 @@
 
 namespace heph::concurrency {
 
+struct SpinnerCallbacks {
+  std::function<void() init_cb = []() {};
+  std::function<void() spin_once_cb = []() {};
```

Comment:
what about the StoppableCallback one? We do not want to support in this case? 
also you are missing:
```suggestion
  std::function<void()> spin_once_cb = []() {};
```

Reply:
fixed